src/webgl/lineFeature.js

```diff
@@ -126,19 +134,19 @@ var webgl_lineFeature = function (arg) {
         strokeOffsetFunc = m_this.style.get('strokeOffset'), strokeOffsetVal,
         miterLimit = m_this.style.get('miterLimit')(data),
         antialiasing = m_this.style.get('antialiasing')(data) || 0,
-        order = m_this.featureVertices(),
+        order = m_this.featureVertices(), orderk0, prevkey, nextkey, offkey,
         orderLen = order.length,
-        posBuf, prevBuf, nextBuf, farBuf, flagsBuf, indicesBuf,
+        posBuf, prevBuf, nextBuf, farBuf, flagsBuf,
```

Comment:
@manthey would it be possible to add some note here on each buffer type specifically the flags buff?

Reply:
thanks